internal/commands/manifest_annotate.go

```diff
@@ -0,0 +1,57 @@
+package commands
+
+import (
+	"github.com/buildpacks/pack/internal/style"
+	"github.com/buildpacks/pack/pkg/client"
+	"github.com/buildpacks/pack/pkg/logging"
+	"github.com/spf13/cobra"
+)
+
+type ManifestAnnotateFlags struct {
+	Architecture string // Set the architecture
+	OS           string // Set the operating system
+	Variant      string // Set the architecture variant
+}
+
+func ManifestAnnotate(logger logging.Logger, pack PackClient) *cobra.Command {
+	var flags ManifestAnnotateFlags
+	cmd := &cobra.Command{
+		Use:     "annotate [OPTIONS] <manifest-list> <manifest>",
+		Short:   "Annotate a manifest list",
+		Args:    cobra.MatchAll(cobra.ExactArgs(2)),
+		Example: `pack manifest annotate paketobuildpacks/builder:full-1.0.0 \ paketobuildpacks/builder:full-linux-amd64`,
```

Comment:
Maybe we can update the examples, to use the same `cnbs/sample-package` as before, in that way we do not confuse the end user